commit 8a3e8de5c0dfbdcf6b8b3317b8f9f10fa3084ad4 (refs/changes/43/2943/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-13T14:46:10-08:00 (1 year, 11 months ago)
    
    PUBAPI-1458 CA tests broken by changes in error objects' structure

diff --git a/test/analytics.test.js b/test/analytics.test.js
index c63a5f3..c9bd8ca 100644
--- a/test/analytics.test.js
+++ b/test/analytics.test.js
@@ -690,6 +690,8 @@ test('DeleteInstrumentation OK', function (t) {
             t.equal(res2.statusCode, 404);
 
             t.deepEqual(err2, {
+                jse_info: {},
+                jse_shortmsg: '',
                 message: 'resource not found',
                 statusCode: 404,
                 restCode: 'ResourceNotFound',
